testsuite/integration/microprofile-tck/rest-client/pom.xml

```diff
@@ -0,0 +1,373 @@
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>uk.co.automatictester</groupId>
+                <artifactId>wiremock-maven-plugin</artifactId>
+                <version>4.1.0</version>
+                <executions>
+                    <execution>
+                        <phase>generate-test-sources</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <dir>target/classes</dir>
+                            <params>--port=8765</params>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>install-as-module-wiremock</id>
+                        <phase>generate-test-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.github.tomakehurst</groupId>
+                                    <artifactId>wiremock</artifactId>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>junit</groupId>
+                                    <artifactId>junit</artifactId>
+                                </artifactItem>
+                            </artifactItems>
+                            <stripVersion>true</stripVersion>
+                            <outputDirectory>${project.build.directory}/wildfly/wiremock</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.org.wildfly.plugin}</version>
+                <executions>
+                    <execution>
+                        <id>add-wiremock-module</id>
+                        <phase>process-test-resources</phase>
+                        <goals>
+                            <goal>execute-commands</goal>
+                        </goals>
+                        <configuration>
+                            <commands>
+                                <command>module add --name=com.github.tomakehurst.wiremock --resources=${project.build.directory}/wildfly/wiremock/wiremock.jar:${project.build.directory}/wildfly/wiremock/junit.jar --module-xml=${basedir}/module.xml</command>
```

Comment:
@asoldano https://github.com/wildfly/wildfly/pull/13208/commits/33957519ca393e5c1740555ac46743acb5038fdb gets past the Windows failure we're seeing in CI on this PR but then the tests fail anyway with what sounds like some problem with the beans.xml deployment overlay.

```
[16:46:01][Step 2/3] [INFO] Running TestSuite
[16:46:08][Step 2/3] [INFO] RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
[16:46:36][Step 2/3] [ERROR] Tests run: 340, Failures: 44, Errors: 0, Skipped: 296, Time elapsed: 35.353 s <<< FAILURE! - in TestSuite
[16:46:36][Step 2/3] [ERROR] arquillianBeforeClass(org.eclipse.microprofile.rest.client.tck.AdditionalRegistrationTest)  Time elapsed: 8.903 s  <<< FAILURE!
[16:46:36][Step 2/3] org.jboss.arquillian.container.spi.client.container.DeploymentException: 
[16:46:36][Step 2/3] Cannot deploy AdditionalRegistrationTest.war: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => {"WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"AdditionalRegistrationTest.war\".STRUCTURE" => "WFLYSRV0153: Failed to process phase STRUCTURE of deployment \"AdditionalRegistrationTest.war\"
[16:46:36][Step 2/3]     Caused by: org.jboss.as.server.deployment.DeploymentUnitProcessingException: WFLYSRV0196: Failed to get content for deployment overlay WEB-INF/lib/*.jar/META-INF/beans.xml at WEB-INF/lib/*.jar/META-INF/beans.xml
[16:46:36][Step 2/3]     Caused by: java.io.FileNotFoundException: C:\\BuildAgent\\work\\f5da3564a57e9d74\\testsuite\\integration\\microprofile-tck\\rest-client\\target\\wildfly\\standalone\\tmp\\vfs\\temp\\tempae0fca84ca4e4199\\content-90ee14a7fedaf8cb\\WEB-INF\\lib\\*.jar\\META-INF\\beans.xml (The filename, directory name, or volume label syntax is incorrect)"}}}}
[16:46:36][Step 2/3] 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:184)
[16:46:36][Step 2/3] 	at org.jboss.as.arquillian.container.ArchiveDeployer.deployInternal(ArchiveDeployer.java:162)
```

Reply:
If this is too much hassle I think just using a profile to turn this off on windows would be ok.